### ISSUE TITLE
Bug- Request body and example missed  when created a second same swagger-ui

### ISSUE THREAD

**Issue by AnthonyHao** [swagger-api/swagger-ui#4712]
### Q&A (please complete the following information)
 - OS: [Win10]
 - Browser: [chrome]
 - Version: [67]
 - Method of installation: [dist]
 - Swagger-UI version: [3.17.0]
 - Swagger/OpenAPI version: [ OpenAPI 3.0]

### Content & configuration

Example Swagger/OpenAPI definition:
```yaml
# your YAML here
# We use JSON, I convert to YAML. You can check the attachment files.
---

openapi: 3.0.0
info:
  version: ''
  title: com.sap.gtt.app.deliverysample.DeliveryWriteService
paths:
  "/DeliveryProcess/deliveryProcess":
    post:
      summary: ''
      operationId: ''
      tags:
      - DeliveryProcess
      responses:
        '200':
          description: Expected response to a valid request
          content:
            application/json:
              schema:
                type: string
        default:
          description: unexpected error
          content:
            application/json:
              schema:
                "$ref": "#/components/schemas/Error"
      requestBody:
        description: ''
        required: true
        content:
          application/json:
            schema:
              "$ref": "#/components/schemas/DeliveryProcess"
components:
  schemas:
    DeliveryProcess:
      properties:
        trackingIds:
          type: array
          items:
            "$ref": "#/components/schemas/QualifiedTrackingId"
        altKey:
          maxLength: 255
          type: string
        processEvents:
          type: array
          items:
            "$ref": "#/components/schemas/ProcessEventDirectory"
        deliveryId:
          maxLength: 10
          type: string
        salesOrderId:
          maxLength: 10
          type: string
        shipToParty:
          type: string
          maxLength: 255
        soldToParty:
          type: string
          maxLength: 255
        salesOrganization:
          maxLength: 4
          type: string
        deliveryDate:
          type: string
          format: date
        loadingDate:
          type: string
          format: date
        transPlanDate:
          type: string
          format: date
        shippingPoint:
          type: string
          maxLength: 255
        deliveryWeight:
          multipleOf: 0.001
          maximum: 999999999999.999
          minimum: -999999999999.999
          type: number
          format: decimal
        deliveryNetWeight:
          multipleOf: 0.001
          maximum: 999999999999.999
          minimum: -999999999999.999
          type: number
          format: decimal
        deliveryWeightUnit:
          maxLength: 6
          type: string
        deliveryVolume:
          multipleOf: 0.001
          maximum: 999999999999.999
          minimum: -999999999999.999
          type: number
          format: decimal
        deliveryVolumeUnit:
          maxLength: 6
          type: string
        shippingType:
          enum:
          - '01'
          - '02'
          - '03'
          - '04'
          - '05'
          - DU
          - LG
          - LT
          - MM
          - P1
          - P2
          - P3
          - WM
          maxLength: 20
          type: string
        purchaseOrderId:
          maxLength: 20
          type: string
        numberOfPackages:
          type: integer
          format: int32
        deliveryStatus:
          enum:
          - '01'
          - '02'
          maxLength: 2
          type: string
        shippingStatus:
          enum:
          - '01'
          - '02'
          - '03'
          - '04'
          maxLength: 50
          type: string
        items:
          type: array
          items:
            "$ref": "#/components/schemas/DeliveryItem"
    QualifiedTrackingId:
      properties:
        altKey:
          maxLength: 255
          type: string
        validFromUTC:
          type: string
          format: timestamp
        validToUTC:
          type: string
          format: timestamp
    ProcessEventDirectory:
      properties:
        eventType:
          maxLength: 255
          type: string
        location:
          type: string
          maxLength: 255
        plannedTechnicalTimestamp:
          type: string
          format: timestamp
        plannedBusinessTimestamp:
          type: string
          format: timestamp
    ObjectReference:
      properties:
        obsType:
          enum:
          - parent
          - child
          maxLength: 50
          type: string
        objKey:
          maxLength: 255
          type: string
    ObserverAddDeleteByBusSysKey:
      properties:
        altKey:
          maxLength: 255
          type: string
        validFromUTC:
          type: string
          format: timestamp
        validToUTC:
          type: string
          format: timestamp
        action:
          enum:
          - A
          - D
          maxLength: 50
          type: string



Swagger-UI configuration options:
```js
SwaggerUI({
 SwaggerUIBundle({
        spec: _getSpec(), //I write a function to get the JSON
        supportedSubmitMethods: [],
        dom_id: '#another-swagger-ui',
        deepLinking: true,
        presets: [
          SwaggerUIBundle.presets.apis,
          SwaggerUIStandalonePreset
        ],
        plugins: [
          SwaggerUIBundle.plugins.DownloadUrl
        ],
        layout: "StandaloneLayout"
      })
})
```

```
?yourQueryStringConfig
```

### Describe the bug you're encountering

I create two swagger-uis in a html page, the same spec, all the configurations are the same. It seems some parameters missed in the second swagger-ui.

### To reproduce...
Open the attachment, you can find it after opening the html file.

1.Download the attachment
2.extract the zip
3.open the index.html

### Screenshots

![image](https://user-images.githubusercontent.com/9319975/42549659-ed382b90-84ff-11e8-9ffa-9d47a3075d68.png)
### 

- 

- _****_

#The red area is parameters missed.
### Additional context or thoughts

I guess some wrong in our yaml(json) file, but we can't find out.
[dist.zip](https://github.com/swagger-api/swagger-ui/files/2182872/dist.zip)

**Comment by zrx8269** [comment]
the same issue +1

**Comment by Raptor399** [comment]
I don't think anything is wrong in your yaml file.

I see the same effect when using VS Code with the arjun-g/vs-swagger-viewer extension. Initially requestBody renders, but when the source is edited and the contents are being re-rendered, requestBody no longer appears. Observe with a browser pointed at http://localhost:9000/index.html while Preview swagger is open and you can reload the page to have requestBody rendered again.

Haven't been able to pinpoint the rendering problem yet.

**Comment by shockey** [comment]
This should be fixed by #4923 - closing as resolved. 

This is simply to keep our issue tracker clean - feel free to comment if there are any further thoughts or concerns, and we'll be happy to reopen this issue.
